src/main/scala/it/pagopa/interop/purposeprocess/service/impl/PDFCreatorImpl.scala

```diff
@@ -123,8 +124,21 @@ object PDFCreatorImpl extends PDFCreator with PDFManager {
       case c: MultiAnswerQuestionConfig  => getMultiAnswerTextFromConfig(c, answer, language)
     }
 
-  private[this] def getSingleAnswerTextFromConfig(answer: RiskAnalysisSingleAnswer): Try[String] =
-    answer.value.toTry(UnexpectedEmptyAnswer(answer.key))
+  private[this] def getSingleAnswerTextFromConfig(
+    questionConfig: SingleAnswerQuestionConfig,
+    answer: RiskAnalysisSingleAnswer,
+    language: Language
+  ): Try[String] =
```

Comment:
Why not convert this to:
```
  private[this] def getSingleAnswerText(
    language: Language
  ): (FormConfigQuestion, RiskAnalysisSingleAnswer) => Try[String] = (questionConfig, answer) =>
```
so you can convert this 
```
formatAnswer(formConfig, language, answer, answer.key, getSingleAnswerText(_, _, language))
```
to
```
formatAnswer(formConfig, language, answer, answer.key, getSingleAnswerText(language))
```

anyway is a minor problem.

Reply:
sure 👍